### PR TITLE
Add Banner and IncompatableBrowserBanner

### PR DESCRIPTION
**Pull request by JasonMHasperhoven**
Addresses #1666 

**Changes**
- Added generic Banner component
- Added IncompatableBrowserBanner component to splash screen and layout (header component)
- Fixed splash screen UI on mobile
- Fixed buttons not to shrink

Before:
image

After:
image

### CODE REVIEW

**Comment by TalDerei** [comment]
can you add a [changeset](https://github.com/penumbra-zone/web/blob/main/docs/documenting-changes.md)?

File: packages/ui/components/ui/testnet-banner/index.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
+import { Banner } from '../banner';
+
 export const TestnetBanner = ({ chainId }: { chainId?: string }) =>
```
Comment (TalDerei) [inline packages/ui/components/ui/testnet-banner/index.tsx:3]: think we can remove `TestnetBanner`

File: packages/ui/components/ui/incompatable-browser-banner/get-compatability.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import { UAParser, UAParserInstance, IDevice } from 'ua-parser-js';
+
+const parser: UAParserInstance = new UAParser();
+
+const CompatableBrowsers: Record<string, number> = {
+  // browser name: min version
+  Chrome: 119,
```
Comment (TalDerei) [inline packages/ui/components/ui/incompatable-browser-banner/get-compatability.ts:7]: Is there a specific reason why Chrome 119 was chosen as the minimum version?

**Review by TalDerei — APPROVED** [review]

File: packages/ui/components/ui/testnet-banner/index.tsx (hunk 1 above)
Comment (JasonMHasperhoven) [reply packages/ui/components/ui/testnet-banner/index.tsx:3]: @grod220 confirm please

File: packages/ui/components/ui/incompatable-browser-banner/get-compatability.ts (hunk 2 above)
Comment (grod220) [reply packages/ui/components/ui/incompatable-browser-banner/get-compatability.ts:7]: Can we add a code comment pointing to that json?

File: packages/ui/components/ui/testnet-banner/index.tsx (hunk 1 above)
Comment (grod220) [reply packages/ui/components/ui/testnet-banner/index.tsx:3]: Nah, let's keep this for now. Think it remains important to somehow distinguish visually mainnet vs testnet.

File: packages/ui/components/ui/incompatable-browser-banner/get-compatability.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+};
+
+interface CompatabilityResponse {
+  isIncompatable: boolean;
+  title: string;
+  content: string;
+}
```
Comment (grod220) [inline packages/ui/components/ui/incompatable-browser-banner/get-compatability.ts:14]: nonblocking: wonder if the type should be something like a rust enum:

```ts
type CompatabilityResponse = { isIncompatable: true } | { isIncompatable: false, title: string, content: string }
```

**Comment by chairulakmal** [comment]
Is IncompatableBrowserBanner is also intended for Brave browser on desktop? I have been using Prax on Brave desktop for a while without any problem.

**Pull request merged**